packages-web/maps/src/components/GoogleMap.tsx

```diff
@@ -58,29 +62,31 @@ export function GoogleMap(props: GoogleMapsProps): ReactElement {
                 map.current.setCenter(bounds.getCenter());
             }
         }
-    }, [map.current, props.locations, props.currentLocation]);
+    }, [map.current, props.locations, props.currentLocation, props.autoZoom]);
 
     return (
         <div className={classNames("widget-maps", props.className)} style={{ ...props.style, ...getDimensions(props) }}>
+            {error && <Alert bootstrapStyle="danger">{error}</Alert>}
             <div className="widget-google-maps-wrapper">
                 <LoadScript
                     googleMapsApiKey={props.mapsToken}
                     id="_com.mendix.widget.custom.Maps.Maps"
                     loadingElement={<div className="spinner" />}
+                    onError={error => setError(error.message)}
                 >
                     <GoogleMapComponent
                         mapContainerClassName="widget-google-maps"
                         options={{
-                            zoomControl,
-                            scrollwheel,
                             draggable,
-                            streetViewControl,
-                            mapTypeControl,
                             fullscreenControl,
-                            rotateControl,
-                            minZoom: 1,
+                            mapTypeControl,
                             maxZoom: 20,
-                            styles: getGoogleMapsStyles(props.mapStyles)
+                            minZoom: 1,
+                            rotateControl,
+                            scrollwheel,
+                            streetViewControl,
+                            styles: getGoogleMapsStyles(props.mapStyles),
```

Comment:
You don't deconstruct `mapStyles` from `props`, while you deconstruct everything else. Maybe don't deconstruct anything?

Reply:
Done